### PR TITLE
test(unit-tests): fix unit test config for gatsby

### PR DESCRIPTION
**Pull request by mario-subo**
- Fixes the config for unit tests since move to gatsby
- fixes the build for production (lib components)
- fixed the build for gatsby (docs)

### CODE REVIEW

File: lib/src/ImageCropper/image-cropper-style.scss [hunk 1]
```diff
@@ -1,4 +1,4 @@
-@import "../../node_modules/cropperjs/dist/cropper.css";
+@import "../../../node_modules/cropperjs/dist/cropper.css";
```
Comment (yousifalraheem) [inline lib/src/ImageCropper/image-cropper-style.scss:1]:
```suggestion
@import "~cropperjs/dist/cropper.css";
```

**Review by kherP — APPROVED** [review]

File: src/components/SideMenu.tsx [hunk 2]
```diff
@@ -15,16 +15,15 @@ const SIDE_MENU_STORAGE_KEY = "SIDEMENU";
 const components: Array<ComponentsListItem> = comps.sort((a, b) => (a.name > b.name ? 1 : -1));
 
 function getStoredValue(): boolean {
-    const value: string = localStorage.getItem(SIDE_MENU_STORAGE_KEY);
+    const w: Window | null = typeof window !== "undefined" ? window : null;
+    const value: string = w === null ? "true" : w.localStorage.getItem(SIDE_MENU_STORAGE_KEY);
```
Comment (yousifalraheem) [inline src/components/SideMenu.tsx:19]: I don't like this solution 😐

Comment (yousifalraheem) [reply src/components/SideMenu.tsx:19]: I understand the need for it, but I feel like the [suggested fix from Gatsby](https://www.gatsbyjs.com/docs/debugging-html-builds/#how-to-check-if-window-is-defined) is way cleaner

Comment (yousifalraheem) [reply src/components/SideMenu.tsx:19]: Check #475 for reference

**Review by yousifalraheem — APPROVED** [review]

**Pull request merged**